session/manager.go

```diff
@@ -27,16 +28,23 @@ type client struct {
 	supported map[string]struct{}
 }
 
+type History struct {
```

Comment:
```suggestion
type SessionHistory struct {
```

Reply:
then again... the package name is `session`, so we'll get `session.SessionHistory`.... ignore me again.

Reply:
Oh yeah, I learned that the hard way already 😄 open to other name suggestions though